bootstrap.py

```diff
@@ -1013,6 +1061,8 @@ def exec_service(service, command, failonerror=True):
     # > Clean the environment from LD_... variables
     clean_environment()
 
+    validate_login()
```

Comment:
Can you add a comment like `# > Validate credentials`, this will ease the generation of the flow as described in https://github.com/Katello/katello-client-bootstrap/blob/master/CONTRIBUTING.md#developer-and-contributor-notes